extensions/core/deployment/src/main/java/org/apache/camel/quarkus/core/deployment/BuildProcessor.java

```diff
@@ -71,6 +52,27 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import io.quarkus.arc.deployment.AdditionalBeanBuildItem;
```

Comment:
Isn't the official ordering `io.*` before `org.*`? We should introduce the `impsort-maven-plugin` to get rid of these problems.

Reply:
[here](https://github.com/apache/camel/blob/master/buildingtools/src/main/resources/camel-checkstyle.xml#L138-L141) the checkstyle configuration for imports so it looks good to me

Reply:
Strange. I think I saw `io.*` before `org.*` in some classes and started respecting it since then.